src/automations/WCW.ts

```diff
@@ -0,0 +1,87 @@
+const cron = require('node-cron');
+import { TextChannel } from 'discord.js';
+import { client } from '../index';
+import { CONSTANTS } from '../constants';
+import { formatISO } from 'date-fns';
+import { scheduledEvent } from '../utils/scheduled-event';
+import { PrismaClient } from '@prisma/client';
+import logger from '../utils/logger';
+
+/**
+ * Water Cooler Wednesdays
+ */
+export const run = () => {
+    const prisma = new PrismaClient();
+
+    cron.schedule('0 * * * MON', async () => {
+        const data = await prisma.wcwTopic.findFirst({
+            where: { used: false },
+        });
+
+        if (!data) {
+            const channel = client.channels.cache.get(
+                CONSTANTS.MODERATOR_CHANNEL_ID,
+            ) as TextChannel;
+            return await channel.send(
+                'All Water Cooler Wednesday topics have been used ❗️ Add a new one with /update-wcw command',
```

Comment:
Minor formatting on the returned message 
```suggestion
                "All Water Cooler Wednesday topics have been used ❗️ Add a new one with `/update-wcw` command",
```